composer.json

```diff
@@ -8,6 +8,6 @@
         }
     },
     "require": {
-        "momentohq/client-sdk-php": "1.6.0"
+        "momentohq/client-sdk-php": "v1.7.0"
```

Comment:
is there a reason why this one has a `v` and the prev one didn't?

Reply:
No, both are acceptable